cmake/modules/FindMaya.cmake

```diff
@@ -331,6 +332,30 @@ if (MAYA_OGSDEVICES_LIBRARY)
 endif()
 message(STATUS "Using Maya Light API Version ${MAYA_LIGHTAPI_VERSION}")
 
+set(MAYA_DEFAULT_MATERIAL_API 0)
+# Grep for the new API in OpenMayaRender. Might be quicker than asking MayaPy.
+find_file(MAYA_OPENMAYARENDER_LIBRARY
+        "${MAYA_DSO_PREFIX}OpenMayaRender${MAYA_DSO_SUFFIX}"
+    HINTS
+        "${MAYA_LIBRARY_DIR}"
+        "${MAYA_LOCATION}"
+    PATH_SUFFIXES
+        lib/
+        bin/
+    DOC
+        "Maya's ${MAYA_LIB} library path"
+    # NO_CMAKE_SYSTEM_PATH needed to avoid conflicts between
+    # Maya's Foundation library and OSX's framework.
+    NO_CMAKE_SYSTEM_PATH
+)
+if (MAYA_OPENMAYARENDER_LIBRARY)
+    file(STRINGS ${MAYA_OPENMAYARENDER_LIBRARY} HAS_DEFAULT_MATERIAL_API REGEX "setDefaultMaterialHandling")
+    if (HAS_DEFAULT_MATERIAL_API)
+        set(MAYA_DEFAULT_MATERIAL_API 1)
+        message(STATUS "This version of Maya has the default material API")
```

Comment:
I always like the log to print the name of the env var as well, so I can ctrl+f in the log to check the value that was used.

Reply:
I added a bit more output at the two key detection moments.